common/focuses/wc_magic_lifestyle_focuses.txt

```diff
@@ -0,0 +1,927 @@
+﻿light_magic_focus_1 = {
```

Comment:
Do you plan to add `can_start_new_lifestyle_tree_trigger` in new lifestyles?

Reply:
If I do that, then AI will not mix lifestyles, which is not desirable, I believe?

Reply:
> If I do that, then AI will not mix lifestyles, which is not desirable, I believe?

Fair.